src/ameba/presenter/rule_presenter.cr

```diff
@@ -1,14 +1,24 @@
 module Ameba::Presenter
   class RulePresenter < BasePresenter
     def run(rule)
-      output.puts
       output_title "Rule info"
-      output_paragraph "%s of a %s severity [enabled: %s]" % {
+
+      info = <<-INFO
+        Name:           %s
+        Severity:       %s
+        Enabled:        %s
+        Since version:  %s
+        INFO
```

Comment:
Couldn't we make use of `String.build` here then pass it to `output_paragraph`?

Reply:
I see no reason to do so.